make/autoconf/flags-other.m4

```diff
@@ -33,16 +33,25 @@ AC_DEFUN([FLAGS_SETUP_ARFLAGS],
   # FIXME: figure out if we should select AR flags depending on OS or toolchain.
   if test "x$OPENJDK_TARGET_OS" = xaix; then
     ARFLAGS="-X64"
-  elif test "x$OPENJDK_TARGET_OS" = xwindows; then
-    # lib.exe is used as AR to create static libraries.
-    ARFLAGS="-nologo -NODEFAULTLIB:MSVCRT"
   else
     ARFLAGS=""
   fi
 
   AC_SUBST(ARFLAGS)
 ])
 
+AC_DEFUN([FLAGS_SETUP_LIBFLAGS],
+[
+    # LIB is used to create static libraries on Windows
```

Comment:
indentation.

Reply:
Bummer, I missed this. :-( I will fix it in https://github.com/openjdk/jdk/pull/17986.